benchmarks/llm/README.md

```diff
@@ -1,15 +1,563 @@
 <!--
 SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
 SPDX-License-Identifier: Apache-2.0
+
 Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
+
 http://www.apache.org/licenses/LICENSE-2.0
+
 Unless required by applicable law or agreed to in writing, software
 distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-Coming soon.
+# LLM Deployment Benchmarking Guide
+
+This guide provides detailed steps on benchmarking Large Language Models (LLMs) using the `perf.sh` and `plot_pareto.py` scripts in single and multi-node configurations. These scripts use [AIPerf](https://github.com/triton-inference-server/perf_analyzer) to collect performance metrics and generate Pareto frontier visualizations.
+
+## Overview
+
+The benchmarking tools in this directory help you:
+
+- **Benchmark LLM deployments** at various concurrency levels
+- **Compare performance** between aggregated and disaggregated serving modes
+- **Generate Pareto plots** to visualize throughput vs latency trade-offs
+- **Evaluate different configurations** (tensor parallelism, data parallelism, etc.)
+
+### Scripts
+
+- **`perf.sh`**: Bash script that runs AIPerf benchmarks across multiple concurrency levels
+- **`plot_pareto.py`**: Python script that generates Pareto efficiency graphs from benchmark results
+- **`nginx.conf`**: NGINX configuration template for load balancing (used for baseline comparisons)
+
+## Prerequisites
+
+> [!Important]
+> At least one 8xH100-80GB node is recommended for the following instructions. Different hardware configurations may yield different results.
+
+1. **Dynamo installed** - Follow the [installation guide](../../README.md#installation) to set up Dynamo
+2. **Model downloaded** - Download the model you want to benchmark:
+   ```bash
+   huggingface-cli download neuralmagic/DeepSeek-R1-Distill-Llama-70B-FP8-dynamic
```

Comment:
Like the rest of our examples, I think it would be good to start with a small model like Qwen/Qwen3-0.6B in the README